packages/components/psammead-podcast-promo/src/components/card-image.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { number, string } from 'prop-types';
+
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+
+const Wrapper = styled.figure`
+  width: 88px;
+  display: inline-block;
+  vertical-align: top;
+  margin: 8px 0 0 8px;
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: 109px;
```

Comment:
Are these from hovering over Zeplin designs?

Reply:
aye

Reply:
Should we use GEL spacing here?

Reply:
Gel spacing is all REM values - I don't think we want the image sizes to change based on font size.  Plus we don't have GEL_SPACING variables that match these sizes anyway :)